src/Runner.Worker/ContainerOperationProvider.cs

```diff
@@ -110,7 +110,7 @@ public async Task RunContainersHealthcheck(IExecutionContext executionContext, L
             {
                 var healthcheck = await ContainerHealthcheck(executionContext, container);
 
-                if (!string.Equals(healthcheck, "healthy", StringComparison.OrdinalIgnoreCase))
+                if (!(string.Equals(healthcheck, "healthy", StringComparison.OrdinalIgnoreCase) || string.IsNullOrEmpty(healthcheck)))
```

Comment:
```suggestion
                if (!string.IsNullOrEmpty(healthcheck) && !string.Equals(healthcheck, "healthy", StringComparison.OrdinalIgnoreCase))
```

Reply:
this is correct but it leaves out the case when the string is empty which we currently have in case there is no healtcheck and can be ignored. I think the complete correct version would be 
`if (!string.IsNullOrEmpty(healthcheck) || !string.Equals(healthcheck, "healthy", StringComparison.OrdinalIgnoreCase))`

Reply:
```suggestion
                var healthyStatuses = new string[] { string.Empty, "healthy" };
                if (!healthyStatuses.Contains(healthcheck, StringComparer.OrdinalIgnoreCase))                
```

Double negatives with '&&' break my head 😵🥴